.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
     "python.linting.enabled": true,
     "python.linting.flake8Enabled": true,
-    "python.linting.flake8Args": [ "--max-line-length=120"]
+    "python.linting.flake8Args": [
+        "--max-line-length=120"
+    ],
+    "python.pythonPath": "env\\Scripts\\python.exe"
```

Comment:
it's better not to commit this part of vscode settings (unlike linter settings, this is individual)